src/Sulu/Bundle/WebsiteBundle/Entity/Analytic.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\WebsiteBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+
+/**
+ * Analytic.
+ */
+class Analytic
```

Comment:
Maybe we can call this one also `Analytics`? Or are there any problems with the plural?